### PR TITLE
Refactor E2E test

### PR DESCRIPTION
**Pull request by orelmisan**
Following PRs #163 and #168, some terminology was changed.
Align the terminology in the E2E test and Makefile to match.

Make the following environment variables mandatory:
- `TEST_NAMESPACE`
- `NETWORK_ATTACHMENT_DEFINITION_NAME`

The following environment variables were renamed:
- `TEST_IMAGE` -> `TEST_CHECKUP_IMAGE`
- `TRAFFIC_GEN_IMAGE_URL` -> `TRAFFIC_GEN_CONTAINER_DISK_IMAGE`
- `VM_CONTAINER_DISK_IMAGE_URL` -> `VM_UNDER_TEST_CONTAINER_DISK_IMAGE`

### CODE REVIEW

File: tests/checkup_test.go [hunk 1]
```diff
@@ -283,19 +283,27 @@ func newRoleBinding(name, serviceAccountName, roleName string) *rbacv1.RoleBindi
 }
 
 func newConfigMap() *corev1.ConfigMap {
+	testConfig := map[string]string{
```
Comment (RamLavi) [inline tests/checkup_test.go:286]: can this commit be in another PR? It's not a refactor, it's more a fix, no?

Comment (orelmisan) [reply tests/checkup_test.go]: Please see PR #181.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/de6e8f55bef3c1ef45590306bef889bf672c1eb0..f8b0539fdd2d1d96f6c484f5500929bf4d048623): Removed last commit.

**Review by RamLavi — APPROVED** [review]
/lgtm
/approve
